drivers/storage/digitalocean/digitalocean.go

```diff
@@ -0,0 +1,47 @@
+// +build !libstorage_storage_driver libstorage_storage_driver_digitalocean
+
+package digitalocean
+
+import (
+	gofigCore "github.com/akutz/gofig"
+	gofig "github.com/akutz/gofig/types"
+)
+
+const (
+	// Name is the name of the driver
+	Name = "digitalocean"
```

Comment:
Hi @protochron,

Does DigitalOcean have any type of abbreviation you'd like to use here? To make configuration properties a little easier/shorter to read/write?